File: Lecture003 If-Else, While loop & Lots of Patterns/Pattern32C.java
package com.company;

import java.util.Scanner;

public class Pattern32C {
    // Pattern 32 C
    public static void main(String [] args){
        /*
        Pattern 32 C
                            E
                            E D
                            E D C
                            E D C B
                            E D C B A
         */
        // Creating Scanner Object For taking inputs from the USER
        Scanner sc = new Scanner(System.in);
        // Take Number of ROWS and COLUMNS from the USER
        System.out.print(" Enter Number of ROWS for the Pattern: ");
        int n = sc.nextInt();
        pattern(n);
    }
    static void pattern(int n){
        // Procedure to print the Required Pattern
        for(int row =1; row <= n ; row++){

            int ch = 60 + n + 4;  // ( e.g E = 69 (in ASCII) so 60 + 5 + 4 )
            for(int column = 1; column <= row ; column++){
                System.out.print( (char)(ch) + " ");
                ch --;
            }
            System.out.println();
        }
    }
}

File: 003_If-Else_While_loop_and_Lots_of_Patterns/Swift/patterns/Pattern32C.swift

import Foundation

// Function to print the specified pattern
func printPattern(_ n: Int) {
    for row in 1...n {
        var ch = Character(UnicodeScalar(60 + n + 4)!)
        for _ in 1...row {
            print("\(ch) ", terminator: "")
            ch = Character(UnicodeScalar(ch.asciiValue! - 1)!)
        }
        print("")
    }
}

// Taking user input for the number of rows
print("Enter Number of ROWS for the Pattern: ", terminator: "")
if let input = readLine(), let n = Int(input) {
    printPattern(n)
} else {
    print("Invalid input")
}
